demo/src/App.js

```diff
@@ -15,6 +15,7 @@ const MIN_SLIDER_VALUE = 0;
 const MAX_SLIDER_VALUE = 65535;
 const MIN_ZOOM = -8;
 const DEFAULT_VIEW_STATE = { zoom: -5.5, target: [30000, 10000, 0] };
+const COLORMAP = 'viridis';
```

Comment:
I think this is fine for demo purposes

Reply:
Agreed; it's fine not to have more options in the demo atm.